.infrastructure/namespace.yml

```diff
@@ -1,4 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: todoapp
\ No newline at end of file
+  name: todoapp
+
+  
```

Comment:
To properly set the EoF is needed one empty line at the end of File
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_4_adding_services/assets/87720583/16f1a3a2-b7c6-429d-9e32-a69550769b85)